packages/bot/src/components/quick-strategy.jsx

```diff
@@ -34,241 +36,315 @@ const TradetypeOption = ({ type }) => (
     </div>
 );
 
-const validateQuickStrategy = values => {
-    const errors = {};
-    const number_field = ['stake', 'size', 'profit', 'loss'];
-
-    Object.keys(values).forEach(key => {
-        const value = values[key];
-
-        if (number_field.includes(key)){
-            if (isNaN(value)) {
-                errors[key] = localize('Must be a number');
-            }
-
-            if (value <= 0){
-                errors[key] = localize('Must be a number higher than 0');
-            }
-
-            if (/^0+(?=\d)/.test(value)) {
-                errors[key] = localize('Invalid number format');
-            }
-        }
-
-        if (value === '') {
-            errors[key] = localize('Field cannot be empty');
-        }
-    });
-
-    return errors;
+const getSizeDesc = index => {
+    switch (index) {
+        case 0:
+            return 'The multiplier amount used to increase your stake if you’re losing a trade.';
+        case 1:
+            return 'The amount that you may add to your stake if you’re losing a trade.';
+        case 2:
+            return 'The amount that you may add to your stake after each successful trade.';
+        default:
+            return '';
+    }
 };
 
-class QuickStrategy extends React.PureComponent {
-    componentDidMount() {
-        this.props.onMount();
+const getSizeText = index => {
```

Comment:
Same here, try to move this logic to the store, the component should only be concerned with presenting the data.